coala_quickstart/coala_quickstart.py

```diff
@@ -54,6 +56,10 @@ def _get_arg_parser():
         dest='no_filter_by_capabilities', const=True,
         help='disable filtering of bears by their capabilties.')
 
+    arg_parser.add_argument(
+        '-g', '--green-mode', const=True, action='store_const',
```

Comment:
Shouldn't the green mode be turned off by default?

Reply:
This is not default value, this is a constant value